web/src/components/molecules/Schema/types.ts

```diff
@@ -0,0 +1,35 @@
+export type Model = {
+  id: string;
+  name: string;
+  description: string;
```

Comment:
Isn't description optional? (In your modalCreationModel PR the form doesn't make description required)